### ISSUE TITLE
run API on my own code

### ISSUE THREAD

**Issue by Jan-990127** [replicate/inpainter#28]
Hi,

Thank you for your good work, is it possible for me to use painting API on my own code, not on the web page?

Best regards

**Comment by zeke** [comment]
Yes! This app actually exists as a reference for people who want to write their own code.

Here's the model it's running: https://replicate.com/stability-ai/stable-diffusion/api

Here's the code that's making requests to Replicate's API: https://github.com/replicate/inpainter/tree/main/pages/api/predictions

Hope that helps!

**Comment by zeke** [comment]
Oh and here's a guide to building your own app with Next.js that uses Replicate's API: https://replicate.com/docs/get-started/nextjs

**Comment by itspranavajay** [comment]
Try out this i guess? https://enhanceai.art/inpainting
